### ISSUE TITLE
[a11y] 11.1.1.1a Non-text content (control elements)

### ISSUE THREAD

**Issue by tschiebel** [owncloud/ios-app#1329]
audit: https://infinite.owncloud.com/s/qSvYucPltEYUhhc

https://github.com/BIK-BITV/BIK-App-Test/tree/main/Pr%C3%BCfschritte/de
https://www.etsi.org/deliver/etsi_en/301500_301599/301549/03.02.01_60/en_301549v030201p.pdf

### Personal Space

- [x]  The "Menu" button has an alternative text, but this is in English and appears to be the name of a variable.
in English and appears to be the name of a variable.
![image](https://github.com/owncloud/ios-app/assets/132444655/38f96788-289a-435c-9758-e5fe0279c58f)

- [x] The "Plus" button ("+") has an alternative text, but it is in English - "Add item".
in English - "Add item". As this version of the app is a German version
a German version, the alternative text should be in German.

- [x] Note: The list of directories and files contains a "More" button in each
list item contains a "More" button. To support screen reader users
users, it would be advisable to display not only the word "More", but also the name of the
name of the directory or file.
![image](https://github.com/owncloud/ios-app/assets/132444655/91a37614-30e1-48f3-9918-2ff82b3e455a)

### Menu, Shared with me, Spaces

- [x] The "Menu" button does have an alternative text, but this is in
English and appears to be the name of a variable.
![image](https://github.com/owncloud/ios-app/assets/132444655/c9e5a115-7fb3-4ca2-9e99-74fd40db3d77)

### Shared with others

- [x] The "Menu" button does have an alternative text, but this is in English
in English and appears to be the name of a variable.
![image](https://github.com/owncloud/ios-app/assets/132444655/06340c4d-8a78-43a7-b278-375e0287e873)

- [x] The button with the arrow pointing to the right does have an alternative text
alternative text, but it is in English and not in German. 
![image](https://github.com/owncloud/ios-app/assets/132444655/70ddda82-7039-46f7-ab88-5a45e00640c6)

### Image viewer

- [x] The "Menu" button has an alternative text, but this is in English and appears to be the name of a variable.
in English and appears to be the name of a variable.
![image](https://github.com/owncloud/ios-app/assets/132444655/625e5fee-b24c-4724-b33a-c3cd269cc539)

- [x] The alternative text of the "Actions" button contains both the word
"Actions" as well as the file name, which makes it difficult to understand.
leads to poor comprehensibility. It is recommended to remove the file name from the alternative text.
from the alternative text.
![image](https://github.com/owncloud/ios-app/assets/132444655/ec2859bb-dd66-4151-aff1-33f04a08686e)

**Comment by jesmrec** [comment]
this is OK from my side.

Just as a NOTE: Translations are not an app feature, i assume that new and existing translations are OK for German language
